### ISSUE TITLE
  Hi. I have been trying to use the 'an...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#5103]
_Submitted from the feedback form on the WormBase website._

<p>&nbsp;&nbsp;Hi. I have been trying to use the 'analyze a list' search feature on wormMine for the last few days and it seems to not be working (it just says "verifying identifiers" but never returns information). I just want to make sure someone is aware of the problem. I use this feature all the time (it is great!) and would like to make sure it gets back online. <br/>Thanks for your help!</p>

**Reported by:** Beck******\* (regr***********_)
*_Submitted from:** <a target="_blank" href="http://www.wormbase.org//tools/support">/tools/support</a>
**Browser:** Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36

**Comment by chris-grove** [comment]
Hi Becky,


 Yes, there is a known issue with the 'analyze a list' feature that the development team is working on. There is a work-around, though. Instead of clicking directly on the "ANALYSE" button on the WormMine homepage (at the bottom of the "Analyze a list" panel), click on the "Lists" tab, then click on the "Upload" sub-tab (if not already selected). From there you can select the entity type (e.g. "Gene"). As for the "for Organism" drop-down, depending on the entity type you may or may not want to specify the species, because this feature is having issues. For example, if you are entering a list of genes, specify the species and click on "Create List". This should work without issue. If you are entering alleles, however, don't specify the species as that seems to throw it off.


So bottom line is, for now:


1) Create a new list to analyze under the "Lists" tab

2) Play with specifying (or not) the species for the list of entities

Hope that helps!


Best,


Chris Grove

WormBase

**Comment by chris-grove** [comment]
I thought this issue already had a ticket, but I can't seem to find it, so I'm going to leave it open.

**Comment by chris-grove** [comment]
This issue appears to be fixed in im-254
